<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportsDialog</class>
 <widget class="QDialog" name="ReportsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>426</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>justagain DIN</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>التقارير</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>815</width>
        <height>336</height>
       </rect>
      </property>
      <attribute name="label">
       <string>المبيعات</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QTableView" name="tableView"/>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>نوع التقرير</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="cash_sell_type">
               <property name="text">
                <string>البيع نقدا</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="later_sell_type">
               <property name="text">
                <string>البيع بالأجل</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="refund_cash_sell_type">
               <property name="text">
                <string>مرودادت نقدا</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="refund_later_sell_type">
               <property name="text">
                <string>مردودات بالأجل</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="consumer_items">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>مستهلك</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="retention_items">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>حجز</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="unretention_items">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>الغاء حجز</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QGroupBox" name="groupBox_6">
                <property name="title">
                 <string>من التاريخ</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <layout class="QGridLayout" name="gridLayout_6">
                 <item row="0" column="0">
                  <widget class="QDateTimeEdit" name="fromdate">
                   <property name="displayFormat">
                    <string>yyyy-MM-dd</string>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                   <property name="timeSpec">
                    <enum>Qt::LocalTime</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_7">
                <property name="title">
                 <string>الي تاريخ</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="0" column="0">
                  <widget class="QDateTimeEdit" name="todate">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="displayFormat">
                    <string>yyyy-MM-dd</string>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>الموظف</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QComboBox" name="user">
                   <property name="styleSheet">
                    <string notr="true">combobox-popup: 0;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>الإجمالي</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                  <widget class="QLineEdit" name="total">
                   <property name="text">
                    <string>0.000</string>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>عرض</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>الغاء</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>745</width>
        <height>336</height>
       </rect>
      </property>
      <attribute name="label">
       <string>العملاء</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="2" column="0">
           <widget class="QTableView" name="tableView_2"/>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>نوع التقرير</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QRadioButton" name="client_later_sell_type">
               <property name="text">
                <string>المبيعات بالأجل</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="client_debt">
               <property name="text">
                <string>الديون</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="client_retention">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>المحجوزات</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QGroupBox" name="groupBox_8">
                <property name="title">
                 <string>من التاريخ</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <layout class="QGridLayout" name="gridLayout_9">
                 <item row="0" column="0">
                  <widget class="QDateTimeEdit" name="fromdate_2">
                   <property name="displayFormat">
                    <string>yyyy-MM-dd</string>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                   <property name="timeSpec">
                    <enum>Qt::LocalTime</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_9">
                <property name="title">
                 <string>الي تاريخ</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <layout class="QGridLayout" name="gridLayout_10">
                 <item row="0" column="0">
                  <widget class="QDateTimeEdit" name="todate_2">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="displayFormat">
                    <string>yyyy-MM-dd</string>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_12">
                <property name="title">
                 <string>العميل</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_15">
                 <item row="0" column="0">
                  <widget class="QComboBox" name="client_name">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">combobox-popup: 0;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_11">
                <property name="title">
                 <string>رقم الفاتورة</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_14">
                 <item row="0" column="0">
                  <widget class="QComboBox" name="bill_num">
                   <property name="styleSheet">
                    <string notr="true">combobox-popup: 0;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_10">
                <property name="title">
                 <string>الإجمالي</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_12">
                 <item row="0" column="0">
                  <widget class="QLineEdit" name="total_2">
                   <property name="text">
                    <string>0.000</string>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>عرض</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="text">
                 <string>الغاء</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>ReportsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
